### ISSUE TITLE
[ plugin/coc.vim ] line  484: E1208: -complete used without -nargs

### ISSUE THREAD

**Issue by xvim64** [neoclide/coc.nvim#3209]
## Describe the bug
```
Error detected while processing C:\HOME\.vim\bundle\coc.nvim\plugin\coc.vim:
line  484: E1208: -complete used without -nargs
```

plugin/coc.vim : 484 : command! -nargs=0 -complete=custom,coc#list#names CocListCancel   :call coc#rpc#notify('listCancel', [])

## Reproduce the bug

_vimrc

```
set bomb
set enc=utf-8
set fenc=utf-8
set fencs=utf-8
set nocompatible
source ~/.vim/bundle/vim-plug/plug.vim
call plug#begin('~/.vim/bundle')
    Plug 'neoclide/coc.nvim', {'branch': 'release'}
call plug#end()
```

**Comment by perrin4869** [comment]
@chemzqm Is there a chance you can merge this fix into the `release` branch? Thanks!!

**Comment by chemzqm** [comment]
@perrin4869 done.

**Comment by murchu27** [comment]
For anyone who comes to this later, upgrading coc.nvim should fix the issue (e.g., with vim-plug, run a simple `PlugUpgrade` within (neo)vim).

**Comment by LeonardChallis** [comment]
> For anyone who comes to this later, upgrading coc.nvim should fix the issue (e.g., with vim-plug, run a simple `PlugUpgrade` within (neo)vim).

Thanks a bunch for this! Quick note: you should run `PlugUpdate` to update plugins, `PlugUpgrade` will update vim-plug itself.

**Comment by murchu27** [comment]
> > For anyone who comes to this later, upgrading coc.nvim should fix the issue (e.g., with vim-plug, run a simple `PlugUpgrade` within (neo)vim).
> 
> Thanks a bunch for this! Quick note: you should run `PlugUpdate` to update plugins, `PlugUpgrade` will update vim-plug itself.

Ah, my bad, I'll edit the original comment

**Comment by blankenshipz** [comment]
I just did a fresh install of `coc.nvim` and I'm seeing this issue; it looks like it plug resolved the tag to `v0.0.80`

**Comment by chemzqm** [comment]
v0.0.80 is too old, use release branch of coc.nvim is recommended.
